ros/lib/config.py

```diff
@@ -25,3 +25,5 @@
 APP_NAME = os.getenv("APP_NAME", "ros")
 
 INSIGHTS_EXTRACT_LOGLEVEL = os.getenv("INSIGHTS_EXTRACT_LOGLEVEL", "ERROR")
+
+GARBAGE_COLLECTION_INTERVAL = os.getenv("GARBAGE_COLLECTION_INTERVAL", 86400)
```

Comment:
Could you please add one sentence about this variable?

Reply:
Added comment in the code.